app/server/src/routes/resource/resource.validate.js

```diff
@@ -21,5 +21,6 @@ exports.validate = (method) => {
         body("resource_id", "resource_id doesn't exist or broken").exists().isMongoId(),
         body("rating", "rating doesn't exist").exists(),
       ];
+    }
```

Comment:
You forgot to add case for get-resource I suppose?

Reply:
Yep, done!